packages/react/package.json

```diff
@@ -9,12 +9,20 @@
   "files": [
     "dist"
   ],
-  "license": "MIT",
+  "license": "Apache-2.0",
   "scripts": {
-    "build": "yarn build:cjs &&yarn build:esm && yarn build:types",
+    "build": "yarn build:clean && yarn build:cjs &&yarn build:esm && yarn build:types",
```

Comment:
I suggest you to look into [`tsup`](https://www.npmjs.com/package/tsup) for the build tool.
It'll make this more readable/maintainable.

Reply:
I really don't want to introduce more stuff right now, if we need to in the future I'll look into it. This is the setup on viem so I'm sure it's good enough for now